app/src/main/res/values-sw/strings.xml

```diff
@@ -10,6 +10,7 @@
   <string name="menu_help">Msaada</string>
   <string name="exploration_activity_title">Kicheza Ugunduzi</string>
   <string name="help_activity_title">Msaada</string>
+  <string name="bottom_sheet_options_menu_close">Funga</string>
```

Comment:
Please don't add strings for other languages, that's a separate process.

Reply:
Removed